tests/wasm/api.rs

```diff
@@ -0,0 +1,137 @@
+use wasm_bindgen_test::*;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen(module = "tests/wasm/api.js", version = "*")]
```

Comment:
Do we need the version thing here?

Reply:
Unfortunately yeah, I've been meaning to relax this in `#[wasm_bindgen]` for awhile, but since the module doesn't start with `"./"` it requires `version` to be present (it's a bit of a hack today)